src/main/java/core/basesyntax/FigureSupplier.java

```diff
@@ -0,0 +1,48 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+class FigureSupplier {
+    private static final Random RANDOM = new Random();
```

Comment:
```suggestion
    private final Random random = new Random();
```
this is debatable but it's not truly constant anyway - Random itself is mutable, let's do it like in ColorSupplier